docs/upgrade-guides/core/3-20-to-3-21.mdx

```diff
@@ -8,6 +8,132 @@ To follow the zero-downtime strategy when upgrading to 3.21, **It is recommended
 Otherwise, you will need to downtime your solution to ensure correct data migration.
 :::
 
+## Strict authorization token validation
+
+Saleor will enforce strict validation of authorization tokens. In previous versions, when Saleor received an invalid token, it would return a `200 OK`response with a query executed as an anonymous user. This behaviour has been changed to return a `401 Unauthorized`. This change may affect your custom apps that rely on the previous behavior, also storefronts that send invalid tokens to Saleor could be affected. If you have any custom logic that relies on receiving a `200 OK` response for invalid tokens, you will need to update it 
```

Comment:
I would be more explicit here. Does it impact on all possible tokens? If yes, then I would explicitly say:

> Saleor will enforce strict validation of authorization tokens (any JWT tokens, any app access token).

It is a detail, but thanks to this everyone is clear, that it has impact on any type of token. 

---
Does it impact on tokens created by OIDCPlugin?  I yes, maybe also worth to mention
Does it also impact on flow, when we recevie the access_token directly from the OAuth provider?  https://docs.saleor.io/api-usage/authentication#oidc-single-sign-on-sso-flow